sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,995 @@
+majority.  If they do not, then withdraws of arbitrary amounts of BTC
+are fulfilled eventually by _redirecting_ Stackers' PoX payouts to fund withdraw
+requests.
+
+To support this, sBTC offers two modes of operation: **Normal mode** and **Recovery mode**.
+
+In **Normal mode**, the sBTC asset is
+backed 1:1 by BTC sent to a wallet controlled by a large fraction of Stackers,
+as measured by the fraction of locked STX they represent.  Each time BTC is
+sent to this wallet (a deposit operation), an equal number of sBTC are
+automatically minted to an address of the sender's choosing.  Stackers respond
+to withdraw requests by sending BTC to the requester's Bitcoin address, thereby
+maintaining the peg.  The act of responding to withdraw requests automatically
+destroys the requester's equivalent amount of sBTC on the Stacks chain.
+
+If the Normal mode encounters a liveness failure for any reason (including loss
+of BTC), the system transitions to a Recovery mode until enough Stackers come
+back online.
+
+In **Recovery mode**, a fraction of the PoX payouts are
+redirected to withdraw requests such that eventually, all outstanding requests
+will be fulfilled, even if the Stackers _never_ come back online.  While
+considerably slower than Normal mode, the design of Recovery mode ensures all
+sBTC can be redeemed so long as the Stacks blockchain and PoX are online.
+
+## Scope
+
+This proposal only describes Normal mode, and leaves Recovery mode to be described
+in a future SIP.
+
+This proposal, if ratified, would be executed by at least two hard forks:  one
+to implement Normal mode, and a follow-on hard fork to implement Recovery mode.
+This SIP is designed to leverage features proposed by [SIP-021][SIP-021-LINK], and can only
+activate if it is ratified beforehand or concurrently with this SIP.
+
+The Recovery mode SIP cannot activate without this SIP first being ratified.
+
+# Specification
+
+The proposed [SIP-021][SIP-021-LINK] upgrade would do away with Stacks forks, which vastly
+simplifies the sBTC design over what would be needed for a fork tolerant system.
+Importantly, miners and Stackers cannot "roll back" sBTC operations, meaning that deposit
+and withdraw operations follow at-most-once semantics. This lack of "fork risk" means
+that the system can support arbitrarily large amounts of BTC deposited and withdrawn,
+assuming that at least 70% of Stackers by weight are trustworthy.
+
+The sBTC system state is materialized in a boot contract called `.sbtc`.  This
+contract, like `.pox-4`, is managed in part by the node itself.  In particular,
+the node writes sBTC state from Bitcoin into the data space of `.sbtc` (such as
+via a private function that only the node can call), so that it can be accessed
+by smart contracts in Stacks.
+
+The following sections describe the Normal mode of operation.  Recovery mode is
+left for future work, but is summarized in a separate section after Normal mode.
+
+## Deposit
```

Comment:
@FriendsFerdinand I wrote this section with a mostly complete understanding of OP_DROP but am missing some details. Please suggest adjustments when you can.

I've based this section on these discussions:
1. https://github.com/stacks-network/sbtc/discussions/314
2. https://github.com/stacks-network/sbtc/discussions/315